### ISSUE TITLE
TS Error Popup Not Showing

### ISSUE THREAD

**Issue by lordlycastle** [lukehaas/RunJS#286]
Below is the snippet I have. It should complain when you assign value to `n`, and it does show the line but it doesn't show the popup.

```ts
import * as fp from 'fp-ts';
import * as mn from 'monet';

// Validation/Either test
type Errors = 'Failed' | 'No Connection';
type SuccessObj = { a?: string; b?: number };
type FailedObj = { c?: string; d?: number };

// W/ fp-ts
let x: fp.either.Either<Errors, SuccessObj> = fp.either.left('Failed');
let y: fp.either.Either<Errors, FailedObj> = fp.either.isLeft(x)
  ? x
  : fp.either.right({ c: 'c', d: 4 });
y;

// W/ monet
let m: mn.Validation<Errors, SuccessObj> = mn.Fail('Failed');
let n: mn.Validation<Errors, FailedObj> = m.isFail()
  ? m
  : mn.Success({ c: 'c', d: 4 });
n;
```

Expected error from [TS playground](https://www.typescriptlang.org/play?#code/JYWwDg9gTgLgBAKjgQwM5wGZk1CI4DkWAtDKgQNwBQoksiK6IAdjnoSBMwKYyVVUA9ILgA1ZABtgAE2QxgXQQFFgMABbcocGN1QwqMAJ5hucJVFxR0AXkIAxZMAndpBOAB9CAOQhwAwlw8AMbyXPxGJnAAygCuQUG6qADyAEYAVnC2AN4oAPwAXHB6UMDMAOYUcCkFcMwxICmacAC+1BGmDk4uqRnZcEE1xaUVcNI1dQ1NrQLCcADqIiRkVM7wAB6FWAB03KoaUFsq6poAPOaWqAA00XEJqMnpAHyZmGA7e5pbzhgwABQEnWcrgAlNRVnBDJs3rtjgcjvszhZoFc4IDuk8XtsYfstsBUAAZbg-X5rYFUOBwXJwNbkuBQ96wrYlMpqP45IKFAhBAjXaSFAAsLVBVEM1CEIgWcE4PH04JAhRYW3EUlkoWYiIu11i8USPWetkVgP+aJBYN4tQVzCVkhkcgU6vOyOuJr1LxAuNQRrJFKpIFplq22ruqF+7M53N5AqF1GY1CAA).
<img width="805" alt="SS 2021-07-27T18 02 38@2x" src="https://user-images.githubusercontent.com/11729541/127188248-c087edbc-532d-4a43-9609-f371e2f226f4.png">

See video below where I tried to get the error to show. 

https://user-images.githubusercontent.com/11729541/127188126-abd23ed2-4e99-42f6-823a-eadaf1e86afb.mp4

**Comment by lukehaas** [comment]
Thanks, I'll look into it 👍

**Comment by lordlycastle** [comment]
I have noticed this happening in other cases as well, where popup refuses to show up. Some times I've noticed a popup show up when my cursor was not near any error. So feel like this is because it can't consistently detect if your cursor is on the error.

Hope that helps!

**Comment by lukehaas** [comment]
This issue will be fixed in the next release.

**Comment by lukehaas** [comment]
@lordlycastle this is now fixed in version 2.2.2
